File: sistema_cpluplus/codigos/UsuarioGeral.cpp
#include "UsuarioGeral.h"

//acesso somente por assistentes e especialistas
void usuarioGeral(Clinica& clinica)
{
	int opcaoEntraOuSai;

	do
	{
		system("cls");
		cout << "\t\tBEM VINDO AO MENU DA CLINICA " << clinica.getNomeDaClinica() << "\n\n";

		cout << "[0]-Sair\n[1]-Agenda\n\nDigite: ";
		lerValorCorretamente(opcaoEntraOuSai);

		switch (opcaoEntraOuSai)
		{
		case 0:
			return;

		case 1:
			//Agenda
			agenda(clinica);

			break;

		default:
			//opcao incorreta

			cout << "\nEscolha Corretamente a opcao !\n\n";
			system("pause");
			break;
		}

	} while (true);
}

File: sistema_java/codigos/oficinapoo/UsuarioGeral.java
package oficinapoo;
import static oficinapoo.LerValorCorretamente.*;

public class UsuarioGeral
{  
    public void usuarioGeral(Clinica clinica)
    {
        int opcaoEntraOuSai;
        String leitura;
        UsuarioAdministrador usuarioAdministrador = new UsuarioAdministrador();
        do
        {
            leitura =  "\t\tBEM VINDO AO MENU DA CLINICA " 
                    + clinica.getNomeDaClinica() + "\n\n"
                    + "[0]-Sair\n[1]-Agenda\n\nDigite: ";
            
            opcaoEntraOuSai = lerInteiroCorretamente(leitura);

            switch (opcaoEntraOuSai)
            {
		case 0:
                    return;

		case 1:
                    //Agenda
                    usuarioAdministrador.agenda(clinica);

                    break;

		default:
                    //opcao incorreta

                    mensagemErro("\nEscolha Corretamente a opcao !\n\n");
                    break;
            }
	} while (true);
    }
    
}
